backend/Core/Features/Posts/Commands/CreatePost/CreatePostCommandHandler.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Domain.Entities;
+using Entities.Repository;
+using Shared.CQRS;
+using Shared.Results;
+
+namespace Features.Posts.Commands.CreatePost;
+
+public class CreatePostCommandHandler: ICommandHandler<CreatePostCommand>
+{
+    private readonly IPostRepository _postRepository;
+
+    public CreatePostCommandHandler(IPostRepository postRepository)
+    {
+        _postRepository = postRepository;
+    }
+
+    public async Task<Result> Handle(CreatePostCommand request, CancellationToken cancellationToken)
+    {
+        var post = new Post()
+        {
+            Body = request.Body,
+            CreatedAt = DateTime.Now,
+            Title = request.Title
+        };
+
+        try
+        {
+            await _postRepository.AddAsync(post).ConfigureAwait(false);
```

Comment:
в ASP.NET нет контекста исполнения, если только его задавать в ручную
```suggestion
            await _postRepository.AddAsync(post);
```